content/en/docs/how-tos/add-jobs-to-testgrid.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Add a job to testgrid"
+date: 2020-10-28T11:14:39-04:00
+draft: false
+---
+
+## What is testgrid?
+Testgrid is an aggregator for job results across many runs, allowing us to display test results over time and thus giving an indication of current health, especially in the face of flaky tests.
+
+## What are release-informing/blocking/broken jobs?
+* `Blocking jobs` are those that signal widespread failure of the platform. These are traditionally the core end-to-end test runs on our major platforms and upgrades from previous versions. 
+* `Informing jobs` are a broader suite that test the variety of enviroments and configurations our customers expect. 
+* `Broken jobs` are those that have a known, triaged failure that prevents their function for a sustained period of time (more than a week).
+
+## How to decide if a job should qualify as a release-informing job?
+Release informing jobs in the [release controller config](https://github.com/openshift/release/tree/master/core-services/release-controller/_releases) get run every time we build a new payload for that release.
```

Comment:
This belongs in the section above, and explains that blocking and informing both get triggered

Reply:
This line is providing the motivation for the next line ie. since release informing jobs are run everytime we build a new payload for that release it needs to satisfy the following conditions. I'll change the wording to avoid confusion.